libse/SubtitleFormats/MacSub.cs

```diff
@@ -68,8 +69,8 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                     switch (expecting)
                     {
                         case Expecting.StartFrame:
-                            // 10  = length of int.MaxValues (2147483647); +1 = if contain '/'
-                            if (line.Length <= 10 + 1 && (CharUtils.IsDigit(line[0]) || line[0] == '/'))
+                            // 10 = length of int.MaxValue (2147483647); +1 if starts with '/'
+                            if (len > 0 && len <= 10 + 1 && (CharUtils.IsDigit(line[0]) || line[0] == '/') && CharUtils.IsDigit(line[len - 1]))
```

Comment:
I caught many empty lines passing the 1st condition which will cause `CharUtils.IsDigit(line[0])` to throw...